requirements.txt

```diff
@@ -0,0 +1 @@
+numpy==1.15.4
```

Comment:
I'd be happier if this wasn't a hard requirement: the generator version is still useful without numpy, and should be usable (and tested) without numpy.

Reply:
I removed the requirements.txt file